index.html

```diff
@@ -3,27 +3,26 @@
   <head>
     <meta charset="utf-8"/>
     <title>Web Chat: Markdown</title>
-    
+
     <script src="https://cdn.botframework.com/botframework-webchat/master/webchat.js"></script>
-    <script src="https://cdn.botframework.com/botframework-webchat/4.2.1-master.14eebbb/webchat-es5.js"></script>
+    <script src="https://unpkg.com/core-js@2.6.3/client/core.min.js"></script>
+    <script src="https://unpkg.com/whatwg-fetch@3.0.0/dist/fetch.umd.js"></script>
     <script src="https://unpkg.com/marked@0.6.0/lib/marked.js"></script>
     <script src="https://unpkg.com/sanitize-html@1.20.0/dist/sanitize-html.min.js"></script>
     <link rel="stylesheet" href="index.css">
-    
+
   </head>
   <body>
-    <div class="wc-header"<span> チャットボット </span></div>
+    <div class="wc-header"><span> チャットボット </span></div>
     <div id="webchat" role="main"></div>
-    
-    <script>
 
-      (async function(){
-        fetch('https://directline.botframework.com/v3/directline/tokens/generate', { 
-                                  method: 'POST',
-                                  headers:{
-                                    'Authorization': 'Bearer YOUR_SECRET_KEY'
-                                    } 
-                                  })
+    <script>

```

Comment:
`async` is not supported. But anyway, it is not used here. We are using `then()` instead of `await` here.